README.md

```diff
@@ -75,8 +73,8 @@ Customize path to iostat executable is also possible by setting environment vari
 To learn more about this plugin and iostat tool, visit:
 
 * [linux iostat tool] (http://linux.die.net/man/1/iostat)
-* [snap iostat unit test](https://github.com/intelsdi-x/snap-plugin-collector-iostat/blob/master/iostat/iostat_test.go)
-* [snap iostat examples](#examples)
+* [Snap iostat unit test](https://github.com/intelsdi-x/snap-plugin-collector-iostat/blob/master/iostat/iostat_test.go)
```

Comment:
Change URL to `iostat/iostat_test.go` to make it links correctly on different branch/fork.